detox/e2e/helper/env.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+module.exports = {
+    siteUrl: process.env.SITE_URL || (process.env.IOS ? 'http://localhost:8065' : 'http://10.0.2.2:8065'),
```

Comment:
I missed to describe about this but yes, you're correct. For `SITE_URL`, it'll add flexibility for CI in case we'll spin up a test server not reachable via localhost. Also, it'll open up ability to run test against remote server like our existing test servers.